doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -1366,6 +1366,8 @@ use of libhtp.
        # Maximum decompressed size with a compression ratio
        # above 2048 (only reachable by LZMA)
        #compression-bomb-limit: 1 Mb
+       # Maximum time spent decompressing a single transaction in usec
+       #decompression-time-limit: 100000
```

Comment:
would like to see a little doc addition besides an example config here. It should explain the why of this setting and the trade offs, plus mention the event that is raised if this is exceeed

Reply:
ok, but I am not sure how to test the style of docs changes...